cmd/set/domain/domain.go

```diff
@@ -84,6 +85,14 @@ func get() (*domainManagement, error) {
 }
 
 func set(domain string) error {
+	Match, err := api.Verify(domain)
```

Comment:
Why we need this?

Reply:
To verify if the user has entered one of the pre-existing domains